Library/Homebrew/cmd/update-report.rb

```diff
@@ -235,22 +236,25 @@ def output_update_report
       EOS
     end
 
-    return if new_repository_version.blank?
+    return if new_tag.blank? || args.quiet?
 
     puts
-    ohai "Homebrew was updated to version #{new_repository_version}"
-    Settings.write "latesttag", new_repository_version
-    if new_repository_version.split(".").last == "0"
+    ohai "Homebrew was updated to version #{new_tag}"
+    new_major_version, new_minor_version = (new_tag.split(".")[0, 2]).map(&:to_i)
+    old_major_version, old_minor_version = (old_tag.split(".")[0, 2]).map(&:to_i) if old_tag.present?
+    if old_tag.blank? || new_major_version > old_major_version || new_minor_version > old_minor_version
       puts <<~EOS
         More detailed release notes are available on the Homebrew Blog:
```

Comment:
```suggestion
        The #{new_tag} release notes are available on the Homebrew Blog:
```

Reply:
I added this as `new_major_version}.#{new_minor_version}.0` (see https://github.com/Homebrew/brew/pull/13482#discussion_r915485336) and also added `#{new_tag}` to the changelog message.